project/cookiecutter.json

```
@@ -78,7 +78,7 @@
       "nl": "Nederlands",
       "fi": "Suomi"
     },
-    "github_organization": "GitHub or GitLab Username or Organization",
+    "github_organization": "GitHub-, GitLab-User (or -Organization) Username from url",
```

The punctuation confuses me.

```suggestion
    "github_organization": "GitHub or GitLab username or organization slug from URL",
```

---

In german the leading and trailing dashes mean to re combine unmentioned combinations from the other terms around. Slug has a negative connotation .-( ;-)

---

Also I think you clicked "Resolve conversation" instead of either the "Commit suggestion" for a single suggestion or "Add suggestion to batch" button for multiple suggestions. I clicked "Unresolve conversation" to restore the suggestion and committed it.